features/layout/footer/footer.module.scss

```diff
@@ -0,0 +1,87 @@
+@use "@styles/color";
+@use "@styles/space";
+@use "@styles/breakpoint";
+@use "@styles/z-index";
+@use "@styles/misc";
+
+.footerContainer {
+  background-color: color.$gray-50;
+  padding: 0.75rem;
+
+  @media (min-width: breakpoint.$desktop) {
+    padding: 0;
+  }
+}
+
+.footer {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  align-self: stretch;
+  gap: 1.5rem;
+  padding: 1.5rem;
+  text-align: center;
+
+  @media (min-width: breakpoint.$desktop) {
+    flex-direction: row;
+    padding: 1.12rem 2rem;
+    margin: 0;
+    gap: 0;
+    text-align: left;
+  }
+}
+
+.versionWrap {
+  width: 10rem;
```

Comment:
I'd try avoiding a fixed width here. It's better to let elements wrap the content or use relative attributes like `width: 100%` or `flex: 1` to make the layout responsive